scripts/setup-services.sh

```diff
@@ -83,15 +83,17 @@ cd services
 echo "Setting up login.webmaker.org"
 cd login.webmaker.org
 cp env.sample .env
-sudo npm install --no-bin-links --loglevel=error # sudo needed for bcrypt permissions
+# Refer to https://github.com/npm/npm/issues/10776#issuecomment-209638410 on why we run npm install twice
+sudo npm install --no-bin-links --loglevel=error || sudo npm install --no-bin-links --loglevel=error # sudo needed for bcrypt permissions
```

Comment:
given that we're vagranted: let's take out as much as we can. If it works in vagrant, that's a good reason to accept it.